assert/_diff.ts

```diff
@@ -16,18 +16,14 @@ interface FarthestPoint {
   id: number;
 }
 
-export const DiffType = {
-  removed: "removed",
-  common: "common",
-  added: "added",
-} as const;
+export const DIFF_TYPES = ["added", "removed", "common"] as const;
 
-export type DiffType = keyof typeof DiffType;
+export type DiffType = typeof DIFF_TYPES[number];
```

Comment:
How about directly define this as `"added" | "removed" | "common"` and removing `DIFF_TYPES`?

Reply:
Good idea. Done.